theurgy.lic

```diff
@@ -256,6 +251,7 @@ class TheurgyActions
   def holy_water?
     DRCI.get_item_safe(@water_holder, @theurgy_supply_container)
```

Comment:
Should line 266 also use `get_item_safe`?

https://github.com/rpherbig/dr-scripts/pull/5390/files#diff-f4e18e3d16b30f3605ba753ee5a27d679c60b459748d8708cb17ac111cc9b877R266

Reply:
Can't hurt to make it all uniform.  Done!